st2api/st2api/controllers/v1/actionexecutions.py

```diff
@@ -445,19 +481,15 @@ def delete(self, exec_id):
 
         try:
             (liveaction_db, execution_db) = action_service.request_cancellation(
-                liveaction_db, get_requester())
+                liveaction_db, requester_user.name or cfg.CONF.system_user.user)
```

Comment:
Same here - I assume you had some specific reason to replace utility function with this line?